README.md

```diff
@@ -1 +1,11 @@
-# ast-canonicalization
\ No newline at end of file
+# ast-canonicalization
+
+
+[![elena-lyulina](https://circleci.com/gh/elena-lyulina/ast-canonicalization/tree/setup.svg?style=shield)](https://app.circleci.com/pipelines/github/elena-lyulina/ast-canonicalization?branch=setup)
+
+This repository contains AST-transformations, that transform AST to canonicalized state. These transformations are applied for the AST, that is parsed by [this](src/main/python/pythonparser-3.py) **python3** parser. Source code is in [this](https://github.com/Varal7/pythonparser) repositoty and [this](https://eth-sri.github.io/py150)  project.
+
+ Transformations, that implemented in this project are mentioned in these articles:
```

Comment:
1. A comma before that again, sorry :)

2. Replace, please, Transformations, that implemented -> Transformations, that **are** implemented

3. And use please **in the following articles** instead of **in these articles**. It looks more in English than in Russian